.github/workflows/docs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       steps:      
         - name: Checkout code
-          uses: actions/checkout@v2
+          uses: actions/checkout@v3
```

Comment:
Forgive my ignorance, but why is this change necessary?

Reply:
Right now, with the old version you're getting a warning about an outdated node.js version. This removes it.